core/src/main/scala/com/nvidia/spark/rapids/tool/analysis/AnalysisUtils.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.tool.analysis
+
+object AnalysisUtils {
```

Comment:
nit: This object stores the metric labels. Can we rename the object to reflect something similar (`SQLAnalysisMetrics` etc)?

E.g.

https://github.com/NVIDIA/spark-rapids-tools/blob/730a05dc7b56750d2805ccb5d3261fe6fa938433/core/src/main/scala/com/nvidia/spark/rapids/tool/views/DataSourceView.scala#L37-L42

Reply:
thanks, updated this.